frontend/app/work_packages/controllers/details-tab-overview-controller.js

```diff
@@ -137,9 +137,49 @@ module.exports = function($scope,
   (function setupWorkPackageProperties() {
     var otherAttributes = WorkPackagesOverviewService.getGroupAttributesForGroupedAttributes('other', $scope.groupedAttributes);
 
-    angular.forEach($scope.workPackage.props.customProperties, function(customProperty) {
-      this.push(customProperty);
-    }, otherAttributes);
+    function getFormat(workPackage, prop, propName) {
+      return workPackage.props[propName] ?
+              workPackage.props[propName].format :
+              prop.type.toLowerCase();
```

Comment:
I am not sure where you are ultimately using the result of this method (for me it ends at line 181, where each "customProperty" is `push`ed into an abyss), however I believe you always want to use `prop.type`. 

`workPackage.props[propName].format` would either give you `plain` or `textile` (me knowing the implementation for custom fields: as of today always `plain`). Opposed to that `prop.type` will give you things like `String`, `Formattable`, `Integer`, `Version`, etc.